pkg/tiles/side/side.go

```diff
@@ -1,164 +1,142 @@
 package side
 
-type Side int64
+type Side uint8
 
 const (
-	None Side = iota
-	Top
-	Right
-	Left
-	Bottom
-
-	Center
-
-	// for farmers
-	TopLeftCorner
-	TopRightCorner
-	BottomLeftCorner
-	BottomRightCorner
-
-	// for fields
-
 	/* Left side of top edge */
-	TopLeftEdge
+	TopLeftEdge Side = 0b1000_0000
```

Comment:
This still could be defined using iota

Reply:
I know, but I thought this would be more explicit and understandable.
The sides Top, Right, Bottom, Left still need to be written as 0b... so having the above ones in the same format makes it more clear which bits correspond to which side